packages/zowe-explorer/src/job/ZoweJobNode.ts

```diff
@@ -143,9 +143,9 @@ export class ZoweJobNode extends ZoweTreeNode implements IZoweJobTreeNode {
                 const procstep = spool.procstep ? spool.procstep : undefined;
                 let newLabel: string;
                 if (procstep) {
-                    newLabel = `${spool.stepname}:${spool.ddname} - ${procstep}`;
+                    newLabel = `${spool["id"]} - ${spool.stepname}:${spool.ddname} - ${procstep}`;
```

Comment:
Could we update the reference to the spool ID to use dot notation for property access? This way it would match how we access the other properties. (lines 146 and 148)

```suggestion
                    newLabel = `${spool.id} - ${spool.stepname}:${spool.ddname} - ${procstep}`;
```